modules/geo-layers/src/mesh-layer/mesh-layer-vertex.glsl.js

```diff
@@ -0,0 +1,78 @@
+export default `#version 300 es
+#define SHADER_NAME simple-mesh-layer-vs
+
+// Scale the model
+uniform float sizeScale;
+uniform bool composeModelMatrix;
+
+// Primitive attributes
+in vec3 positions;
+in vec3 normals;
+in vec3 colors;
+in vec2 texCoords;
+
+// Instance attributes
+in vec3 instancePositions;
```

Comment:
Do you need instancing for i3s?

Reply:
It is here because the class inherits `SimpleMeshLayer`

Reply:
You are not using the parent layer model. You are creating your own model using your own shaders. Do not use instanced attributes if you don't need them.

Reply:
Can you remove the instanced attributes?

Reply:
Done